Streaming_Devices/Apple/Apple_TV_Gen2.ir

```diff
@@ -1,8 +1,6 @@
 Filetype: IR signals file
 Version: 1
 # 
-# Apple TV Gen 2
-#
```

Comment:
Any reason why you removed this comment?

Reply:
Not intentionally, the version i had didn't have the comment. After exploring the actual function of the command I want to relabel it anyway so I'll just make a new PR